lib/views/pages/login_signup/register_form.dart

```diff
@@ -528,3 +524,9 @@ class RegisterFormState extends State<RegisterForm> {
     });
   }
 }
+
+enum From {
```

Comment:
How about moving this `enum` to the top ?

Reply:
Though of keeping all enums in the project in a single file.